### ISSUE TITLE
[BUG] - Card

### ISSUE THREAD

**Issue by JsIqbal** [nextui-org/nextui#1221]
### Describe the bug

Module '"@nextui-org/react"' has no exported member 'CardFooter'.

### Your Example Website or App

just import this code and install the package and see for yourself

### Steps to Reproduce the Bug or Issue

import React from "react";
import {Card, CardFooter, Image, Button} from "@nextui-org/react";

export default function App() {
  return (
    <Card
      isFooterBlurred
      radius="lg"
      className="border-none"
    >
      <Image
        alt="Woman listing to music"
        className="object-cover"
        height={200}
        src="/images/hero-card.jpeg"
        width={200}
      />
      <CardFooter className="justify-between before:bg-white/10 border-white/20 border-1 overflow-hidden py-1 absolute before:rounded-xl rounded-large bottom-1 w-[calc(100%_-_8px)] shadow-small ml-1 z-10">
        <p className="text-tiny text-white/80">Available soon.</p>
        <Button className="text-tiny text-white bg-black/20" variant="flat" color="default" radius="lg" size="sm">
          Notify me
        </Button>
      </CardFooter>
    </Card>
  );
}

### Expected behavior

Did not expect the bug 

### Screenshots or Videos

![image](https://github.com/nextui-org/nextui/assets/110350034/25e5086d-42c8-43f4-a1e6-92067aa39f92)

### Operating System Version

windows

### Browser

Chrome

**Comment by johannpayer** [comment]
The name of the component is `Card.Footer`, not `CardFooter`. You only need to import `Card` from `@nextui-org/react`.

**Comment by JsIqbal** [comment]
No..there is a major issue regarding using the next13 + nextUI. I have
resolved the issue in my own way.

this is the documentation:

https://github.com/JsIqbal/portfolio

in the readme.md file you can see the instruction.. because nextui version
2 doesn't work
